File: backend/OrderService/src/main/java/levin/schnabel/orderservice/structural/entity/PaymentMethod.java
package levin.schnabel.orderservice.structural.entity;

public enum PaymentMethod {

	SECURE_PAY("securePay"),
	CREDIT_CARD("creditCard");

	private final String name;

	PaymentMethod(String name) {
		this.name = name;
	}

	public String getName() {
		return name;
	}

}

File: frontend/SportsVendor/src/shared/model/PaymentMethod.ts
export interface PaymentMethod {
  method: 'securePay' | 'creditCard';
  name: string;
}

export const paymentMethods: PaymentMethod[] = [
  {method: 'securePay', name: 'Secure pay'},
  {method: "creditCard", name: 'Credit Card'}
]
